Cannot find type definition file for 'packages/angular_devkit/build_angular/src/typings'.  1 /// <reference types="packages/angular_devkit/build_angular/src/typings" />

ERROR in node_modules/@angular-devkit/build-angular/src/utils/process-bundle.d.ts:1:23 - error TS2688: Cannot find type definition file for 'packages/angular_devkit/build_angular/src/typings'.  1 /// <reference types="packages/angular_devkit/build_angular/src/typings" />

I have this issue. When i check on the file. There is some content inside that file which is commented 

File path
`WebUIa10/node_modules/@angular-devkit/build-angular/src/utils/process-bundle.d.ts`

Content
```
/// <reference types="packages/angular_devkit/build_angular/src/typings" />
/**
 * @license
 * Copyright Google Inc. All Rights Reserved.
 *
 * Use of this source code is governed by an MIT-style license that can be
 * found in the LICENSE file at https://angular.io/license
 */
```

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -49,6 +49,7 @@
     "suppressTsconfigOverrideWarnings": true
   },
   "exclude": [
+    "packages/angular_devkit/build_angular/src/typings.d.ts",
     "bazel-out/**/*",
     "dist/**/*",
     "dist-schema/**",

